source/_posts/generating-pdf-in-ruby-on-rails.md

```diff
@@ -0,0 +1,197 @@
+---
```

Comment:
Give link in content and link to previous content or blog

Reply:
Done